src/main/java/org/opentripplanner/routing/alertpatch/AlertCause.java

```diff
@@ -0,0 +1,55 @@
+package org.opentripplanner.routing.alertpatch;
+
+/**
+ * An enum containing cause options for an alert.
+ */
+public enum AlertCause {
```

Comment:
Is creating these duplicate enums for the internal model wanted?

Reply:
We decided that it's better to use duplicate enums for the internal data models instead of using enum directly from the GTFS RT library.